Library/Homebrew/utils/lock.sh

```diff
@@ -0,0 +1,40 @@
+# Ref: http://wiki.bash-hackers.org/howto/mutex
```

Comment:
There is no mention of license on that website. However, I think I have somehow reimplemented it on my own.